scipy/io/netcdf.py

```diff
@@ -987,6 +983,50 @@ def __setitem__(self, index, data):
                 self.data.resize(shape)
         self.data[index] = data
 
+    def _get_missing_value(self):
+        """
+        Returns the value denoting "no data" for this variable.
+
+        If this variable does not have a missing/fill value, returns None.
```

Comment:
Actually, I see that this reverses the decision previously made in ``__setitem__``, which gave precedence to ``missing_value``. Conceivably this can give issues for users. It may still make sense to change this, but it should be noted in the 0.18.0 release notes I think.